include/bluetooth/gatt.h

```diff
@@ -982,6 +982,9 @@ struct bt_gatt_notify_params {
 	bt_gatt_complete_func_t func;
 	/** Notification Value callback user data */
 	void *user_data;
+#ifdef CONFIG_BT_EATT
+	enum bt_att_bearer_option bearer_option;
+#endif /* CONFIG_BT_EATT */
```

Comment:
Suggest to make this field a generic option, similar to `bt_le_adv_param.options`. That way it can more easily be expanded if we ever choose to add any other option fields.